packages/@featherds/menu/src/components/FeatherMenu.vue

```diff
@@ -73,11 +78,13 @@ export default defineComponent({
     const menu = ref() as Ref<HTMLElement>;
     const open = toRef(props, "open");
     const noExpand = toRef(props, "noExpand");
+    const refId = toRef(props, "dataRefId");
```

Comment:
Why do we need this ref? Can we not use the prop directly in the template?

Reply:
Resolved in latest